gem/management/commands/create_new_banner_link_page_relations.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
+from molo.core.models import ArticlePage, BannerPage, Main
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        for main in Main.objects.all():
+            for banner in BannerPage.objects.descendant_of(main):
+                old_article_slug = ArticlePage.objects.get(
+                    pk=banner.banner_link_page).slug
+                new_article = ArticlePage.objects.descendant_of(
+                    main).get(slug=old_article_slug)
+                banner.banner_link_page = new_article
```

Comment:
@codiebeulaine we need to check here to see if the new_article exist and if not it should remove the banner_link_page and show a warning message